com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameProjectBuilder.java

```diff
@@ -569,6 +569,16 @@ public void build(Task<Void> task) throws CompileExceptionError, IOException {
                 project.getPublisher().AddEntry(liveupdateManifestFilename, manifestTmpFileHandle);
                 project.getPublisher().Publish();
 
+                // Copy SSL public keys if specified
+                String sslCertificatesPath = project.getProjectProperties().getStringValue("network", "ssl_certificates");
+                if (sslCertificatesPath != null && !sslCertificatesPath.isEmpty())
+                {
+                    File source = new File(project.getRootDirectory(), sslCertificatesPath);
+                    File buildDir = new File(project.getRootDirectory(), project.getBuildDirectory());
+                    File dist = new File(buildDir, "ssl_keys.pem");
```

Comment:
I think you should use the constant variable from BundleHelper for the name here?